pkg/asset/cluster/tfvars.go

```diff
@@ -425,6 +431,11 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			Data:     data,
 		})
 	case baremetal.Name:
+		ignitionURL := &url.URL{
+			Scheme: "https",
+			Host:   net.JoinHostPort(installConfig.Config.Platform.BareMetal.APIVIP, "22623"),
+			Path:   "config/master",
+		}
```

Comment:
Can we get this from `masterIgnAsset` rather than recreating it here?

Reply:
+1. That makes sense. Will look into it. Thanks.